cmd/docker-notation/crypto/service.go

```diff
@@ -0,0 +1,40 @@
+package crypto
+
+import (
+	"crypto/x509"
+
+	"github.com/docker/libtrust"
+	"github.com/notaryproject/notation-go-lib"
+	x509n "github.com/notaryproject/notation-go-lib/signature/x509"
+	"github.com/notaryproject/notation-go-lib/simple"
```

Comment:
what is the "simple" module for?

Reply:
This simple module simplifies the process of obtainning a signing service. This module will be removed in the next major iteration of `notation-go-lib`.